lighthouse-core/audits/time-to-interactive.js

```diff
@@ -121,7 +121,7 @@ class TTIMetric extends Audit {
         // Grab this latency and try the threshold again
         currentLatency = estLatency;
       }
-      const timeToInteractive = startTime.toFixed(1);
+      const timeToInteractive = parseFloat(startTime.toFixed(1));
```

Comment:
this is slightly horrifying (slightly more that it was working. thx JS). While you're in here, can you change it to pass `startTime` itself into `distribution.computeComplementaryPercentile()` instead of the rounded `timeToInteractive`?